internal/ethapi/api.go

```diff
@@ -723,6 +723,9 @@ func applyMessage(ctx context.Context, b Backend, args TransactionArgs, state *s
 		}
 		if gp.Gas() < gomath.MaxUint64-postingGas {
 			gp.AddGas(postingGas)
+		} else {
+			log.Warn("gas pool overflow", "gpGas", gp.Gas(), "baseFee", header.BaseFee, "postingGas", postingGas, "msgDataLen", len(msg.Data))
```

Comment:
log.Warn seems to severe. This will happen

Reply:
updated to debug